lib/widgets/nutrition/nutritional_plan_detail.dart

```diff
@@ -274,7 +291,8 @@ class NutritionDiaryEntry extends StatelessWidget {
                     arguments: NutritionalDiaryArguments(plan, date),
                   ),
               child: Text(
-                DateFormat.yMd(Localizations.localeOf(context).languageCode).format(date),
+                DateFormat.yMd(Localizations.localeOf(context).languageCode)
+                    .format(date),
```

Comment:
Can you set the max line length to 100? Then we can get of these diffs (or you just do `flutter format --line-length=100 .` once)

Reply:
No problem